src/services/enrollment.ts

```diff
@@ -0,0 +1,30 @@
+import { getInstance } from '@/configs/axios'
+import { Enrollment } from '@/schemas/enrollment'
+
+const enrollmentMS = getInstance('enrollment')
```

Comment:
getInstance(*name*)

name should be consistent with environment variable

{{name}}_MS_BASE_URL